components/range-list.css

```diff
@@ -39,4 +39,8 @@
 }
 .list-square{
     list-style: square;
-}
\ No newline at end of file
+}
+.notification-list-link .card-horizontal{
+    color: var(--font-dark);
+    border: 1px solid black;
```

Comment:
variable can be used here for black color

Reply:
Yup, fixed it